website/content/docs/secrets/kmip-profiles.mdx

```diff
@@ -171,8 +171,72 @@ Vault implements version 1.4 of the following Key Management Interoperability Pr
+    | i. RSA                                                                               | ✅        | Vault 1.13 |
+    | Object Type [KMIP-SPEC 9.1.3.2.12][kmip-spec-9.1.3.2.12] with value:                 |           |        |
+    | i. Public	 Key                                                                       | ✅        | Vault 1.13 |
+    | ii. Private	 Key                                                                     | ✅        | Vault 1.13 |
+    | Key Format Type [KMIP-SPEC 9.1.3.2.3][kmip-spec-9.1.3.2.3] with value:               |           |        |
+    | i. PKCS#1                                                                            | ✅        | Vault 1.13 <br/> Supported for Private and Public Keys|
+    | ii. PKCS#8                                                                           | ✅        | Vault 1.13 <br/> Supported for Private Key|
+    | iii. Transparent RSA Public Key                                                      | ✅        | Vault 1.13 |
+    | iv. Transparent RSA Private Key                                                      | ✅        | Vault 1.13 |
+
+  6. MAY support any clause within [KMIP-SPEC][kmip-spec] provided it does not conflict with any other clause within the section [Symmetric Key Lifecycle Server][lifecycle-server]
+  7. MAY support extensions outside the scope of this standard (e.g., vendor extensions, conformance clauses) that do not contradict any KMIP requirements.
+
+## [Advanced Cryptographic Server][advanced-cryptographic-server]
+  1. SHALL conform to the [Baseline Server][baseline-server]
+  2. Supports the following client-to-server operations:
+
+    | Operation                                             | Supported | Notes   |
+    | ------------------------------------------------------| :--------:| --------|
+    | Encrypt [KMIP-SPEC 4.29][kmip-spec-4.29]              | ✅        | Vault 1.11 <br/> Supported for AES, unsupported for 3DES: <br/><br/> Supported Block Cipher Modes: <br/> <ol> <li> GCM </li> <li> CBC </li> <li> CFB </li> <li> CTR </li> <li> ECB </li> <li> OFB </li> </ol> <br/> Stream operations are supported except for GCM block cipher mode. <br/><br/> Supported padding methods: <br/> <ol> <li> None </li> <li> PKCS5 </li> </ol> <br/> Vault 1.13 <br/> Supported for RSA Asymmetric Keys  |
+    | Decypt [KMIP-SPEC 4.30][kmip-spec-4.30]               | ✅        | Vault 1.11 <br/> Supported for AES, unsupported for 3DES: <br/><br/> Supported Block Cipher Modes: <br/> <ol> <li> GCM </li> <li> CBC </li> <li> CFB </li> <li> CTR </li> <li> ECB </li> <li> OFB </li> </ol> <br/> Stream operations are supported except for GCM block cipher mode. <br/><br/> Supported padding methods: <br/> <ol> <li> None </li> <li> PKCS5 </li> </ol> <br/> Vault 1.13 <br/> Supported for RSA Asymmetric Keys |
+    | Sign [KMIP-SPEC 4.31][kmip-spec-4.31]                 | ✅        | Vault 1.13 |
```

Comment:
Please add the details for Sign and Signature Verify:
```
// NewSignOperator returns a CryptoOperator to perform a Sign or Signature Verify operation.
//
// Only signing with RSA keys and padding methods PSS and PKCS1v15 is supported.
//
// The supported hashing algorithms with PSS are:
//   - SHA224
//   - SHA256
//   - SHA384
//   - SHA512
//   - RIPEMD160
//   - SHA512_224
//   - SHA512_256
//   - SHA3_224
//   - SHA3_256
//   - SHA3_384
//   - SHA3_512
//
// The supported hashing algorithms with PKCS1v15 are:
//   - SHA224
//   - SHA256
//   - SHA384
//   - SHA512
//   - RIPEMD160
```
See https://github.com/hashicorp/go-kmip/blob/764667b8f9a94ee2eaaf2400b76dd504d66d4678/cryptography/sign_operator.go#L15

Reply:
Hmm, the comment doesn't say, but streaming operations are supported for Sign and Signature Verify.